dart/lib/src/transport/http_transport.dart

```diff
@@ -118,9 +114,9 @@ class _CredentialBuilder {
   }
 
   static String _buildAuthHeader({
-    String publicKey,
-    String secretKey,
-    String sdkIdentifier,
+    String? publicKey,
+    String? secretKey,
+    String? sdkIdentifier,
```

Comment:
wondering if it makes sense to be nullable, a broken header wont be any helpful

Reply:
secret seems to be nullable, the others are not